baselines/getting_started/aws/aws_check_stack/aws_account_iam_stack.tf

```diff
@@ -22,10 +25,8 @@ resource "turbot_policy_setting" "aws_account_iam_stack_source" {
     SOURCE
```

Comment:
this one looks like it cannot run, did you try to terraform apply this?

Reply:
Yes this was tested before the commit and the output/screenshots were attached in this PR aswell.